### PR TITLE
chore: change duckdb wasm package

### PR DESCRIPTION
**Pull request by vpbs2**
Use devrev-duckdb wasm package in dbm and runner
work-item: https://app.devrev.ai/devrev/works/ISS-115803

### CODE REVIEW

File: meerkat-browser-runner/src/app/duck-db/duck-db-bundle-manager.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    }
+    return await this.bundleResolverPromise;
+  }
+}
+
+export const DuckDbBundleManagerInstance = new DuckDbBundleManager({
+  eh: {
+    mainModule: duckdb_wasm_eh,
+    mainWorker: new URL(
+      '@devrev/duckdb-wasm/dist/duckdb-browser-eh.worker.js',
```
Comment (itsTalwar) [inline meerkat-browser-runner/src/app/duck-db/duck-db-bundle-manager.ts:40]: Can we import from some CDN

Comment (vpbs2) [reply meerkat-browser-runner/src/app/duck-db/duck-db-bundle-manager.ts]: This file can be removed

File: meerkat-dbm/package.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "@devrev/meerkat-dbm",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "dependencies": {
     "tslib": "^2.3.0",
-    "@duckdb/duckdb-wasm": "^1.28.0",
+    "@devrev/duckdb-wasm": "^1.14.3",
```
Comment (ujaval403) [inline meerkat-dbm/package.json:6]: Can we remove the auto upgrade for the minor release as well.
We might want to check and update here

Comment (ujaval403) [reply meerkat-dbm/package.json:6]: Same at other package.json as well

**Review by itsTalwar — APPROVED** [review]

**Pull request merged**